src/heuristicblocking.js

```diff
@@ -211,6 +211,13 @@ HeuristicBlocker.prototype = {
       snitch_map.setItem(tracker_origin, firstParties);
     }
 
+    var action_map = this.storage.getBadgerStorageObject('action_map');
+    // If tracker_origin is not yet in action_map, add it with action `constants.ALLOW`
+    if (!action_map.hasItem(tracker_origin)) {
```

Comment:
@cooperq @alexristich Should this be `tracker_fqdn` instead? Otherwise we never initialize FQDNs past the first for an origin ...